README.md

```diff
@@ -131,12 +131,11 @@ repositories {
 
 dependencies {
   implementation platform("io.opentelemetry:opentelemetry-bom:1.1.0-SNAPSHOT")
-  implementation('io.opentelemetry:opentelemetry-api')
+  implementation('io.opentelemetry:opentelemetry-sdk')
 }
 ```
 
-Libraries will usually only need `opentelemetry-api`, while applications
-may want to use `opentelemetry-sdk`.
+Libraries will usually only need `opentelemetry-api`, so should not depend on `opentelemetry-sdk`.
```

Comment:
I like this picture https://github.com/open-telemetry/opentelemetry-specification/blob/main/internal/img/long-term-support.png maybe we can reuse it to show when to depend on which artifact